flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -87,6 +101,17 @@ function setup_kubernetes_for_linux {
     sudo sysctl fs.protected_regular=0
 }
 
+function download_circtl {
```

Comment:
Out of curiosity: Is there a reason why you moved this logic into its own function. You didn't do it for the other two artifacts..